src/sdx/pce/utils/functions.py

```diff
@@ -79,7 +79,9 @@ def biconnectivity(self):
         pass
 
 
-# use dijsktra to get the primary shortest path
+""" use dijsktra to get the primary shortest path """
+
+
 def dijnew(graph, start_node, end_node):
```

Comment:
Docstrings should be below the method name:

```suggestion
def dijnew(graph, start_node, end_node):
    """ use dijsktra to get the primary shortest path """
```